charts/zora/values.yaml

```diff
@@ -221,6 +230,9 @@ scan:
         limits:
           cpu: 500m
           memory: 500Mi
+      # -- Annotations added to the popeye pods
+      podAnnotations:
+        popeye-annotation: popeye-value
```

Comment:
```suggestion
        #popeye-annotation: popeye-value
```

Reply:
I already tried this, but it caused the README to be generated as a string.  Leaving as is for now.